packages/components/Text.tsx

```diff
@@ -0,0 +1,15 @@
+import type { ReactNode } from "react";
+
+interface TextProps {
```

Comment:
I think we are using mainly types over interfaces. Let's rewrite the interfaces as types for now.

Also, as this is a localized props to the `Text.tsx` component, calling it `TextProps` might be redundant, as it's not used elsewhere. I'd suggest renaming it to`Props`.

Reply:
Made in: https://github.com/podcodar/webapp/pull/173/commits/d92a874a6633f957b07f6b0885bb31d183ac8772